client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-overview/hooks/use-stored-cards.ts

```diff
@@ -0,0 +1,54 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useEffect, useState } from 'react';
+import { PaymentMethod } from 'calypso/jetpack-cloud/sections/partner-portal/payment-methods';
+
+// FIXME: Need to hook this up to the real API.
+export default function useStoredCards() {
+	const showDummyData = isEnabled( 'a4a/mock-api-data' );
+
+	const [ isFetching, setIsFetching ] = useState( showDummyData );
+
+	useEffect( () => {
```

Comment:
Add above `// FIXME: Don't forget to remove this fake loading delay for dummy data`